### PR TITLE
updated FAQ with package update time expectations

### PR DESCRIPTION
**Pull request by mrandrewandrade**

### CODE REVIEW

**Comment by ocefpaf** [comment]
I believe that file you edited is auto-generated. I guess that the right one is: https://github.com/conda-forge/conda-forge.github.io/blob/master/src/user/faq.rst

**Comment by mrandrewandrade** [comment]
fixed

File: docs/_sources/user/faq.rst.txt [hunk 1]
```diff
@@ -1,30 +0,0 @@
-FAQ
```
Comment (scopatz) [inline docs/_sources/user/faq.rst.txt]: Please revert this. No need to remove

**Comment by mrandrewandrade** [comment]
Sorry I misunderstood.  Stupid question @scopatz, do I need to manually update that text file or is there a way for me to auto generate it?

**Comment by scopatz** [comment]
No such thing as stupid questions :).  That file is automatically generate by our CI (which runs sphinx).  You don't need to touch anything in that directory at all. Only the `src/` dir really matters for editing.

**Pull request merged**